pkg/cmd/cli/cli.go

```diff
@@ -64,8 +50,8 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 
 	templates.ActsAsRootCommand(cmds, filters, groups...).
-		ExposeFlags(getCmd, "server", "client-certificate",
-			"client-key", "certificate-authority", "insecure-skip-tls-verify", "token")
+		ExposeFlags(groups[0].Commands[0], "server", "client-certificate",
```

Comment:
I don't think so. This is just like a Python index into a list.  The CommandGroups object is a list of CommandGroup objects, which contain a list of Commands.

Hmm, there is another way .... mutiple return values.  Good idea!